packages/layout/src/dimension-estimators/reactflow-dimension-estimator.ts

```diff
@@ -11,9 +12,9 @@ export class ReactflowDimensionsEstimator implements NodeDimensionQueryHandler {
     getWidth(estimatedNode: INodeClassic): number {
         const WIDTH_OF_EMPTY_ATTR = 10;
         // Not using actual model ID so this is just approximation - whole method is just approximation anyways, so it doesn't matter that much
-        const TEST_MODEL_STRING = "https://my-model-7tgfl.iri.todo.com/entities/";
-        const TEST_STRING = TEST_MODEL_STRING + "creepy-office";
-        const APPROXIMATION_OF_WIDTH_OF_ONE_CHARACTER = 359 / TEST_STRING.length;
+        const TEST_MODEL_STRING = "https://my-model-6d9lx.iri.todo.com/entities/";
```

Comment:
Why is this changing?

Reply:
I changed it, because this was for the old version of editor, so I was not sure if the font is of same size as before, etc.